dotnet/src/dotnetcore/Providers/Messaging/GXAzureQueue/GXAzureQueue.csproj

```diff
@@ -1,7 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <PackageId>GeneXus.Azure.Queue</PackageId>
+    <PackageTags>Message Azure</PackageTags>
```

Comment:
Maybe "Azure Queue Messaging" for better discoverability?

Reply:
Done.